lib/connection-manager.js

```diff
@@ -1,125 +1,90 @@
 'use strict'
 
 // External Modules
-const Agent = require('agentkeepalive')
 const async = require('async')
 const debug = require('debug')('logdna:lib:connection-manager')
 const request = require('request')
-const url = require('url')
 
 // Internal Modules
-const fileUtils = require('./file-utilities')
-const getProxyFromURI = require('./getProxyFromURI')
-const linebuffer = require('./linebuffer')
-const utils = require('./utils')
-
-// Variables
-var firstrun
+const client = require('./logger-client.js')
+const fileUtils = require('./file-utilities.js')
+const utils = require('./utils.js')
 
 module.exports = {
-  connectLogServer
+  checkHealth
```

Comment:
Probably don't need to export this because it's coming out soon anyway.  Normally, I'd export it and test it directly to make sure it's switching the log host, but that test will go in the client eventually.

Reply:
and what is preventing us removing the `checkHealth` from the code base, @darinspivey?

Reply:
Removed entirely!